src/site/content/en/accessible/color-and-contrast/index.md

```diff
@@ -0,0 +1,153 @@
+
+<figure class="w-figure">
+  <img src="./contrast-ratios.jpg" alt="An image showing the different contrast ratios">
+</figure>
+
+The contrast ratio of 4.5:1 was chosen for level AA
+because it compensates for the loss in contrast sensitivity
+usually experienced by users with vision loss equivalent to approximately 20/40 vision.
+20/40 is commonly reported as typical visual acuity of people at about age 80.
+For users with low vision impairments or color deficiencies,
+we can increase the contrast up to 7:1 for body text.
+
+You can use the Accessibility Audit in Lighthouse to check your color contrast.
+Open DevTools, click Audits, and select Accessibility to run the report.
+
+<figure class="w-figure">
+  <img src="./accessibility-audit.jpg" alt="An image showing the different contrast ratios">
+</figure>
+
+For a more complete report, install the [Accessibility Insights Extension](https://accessibilityinsights.io/).
+One of the checks in the Fastpass report, is color contrast.
+You will get a detailed report of any failing elements.
+
+<figure class="w-figure">
+  <img src="./fastpass-contrast.jpg" alt="The report in Accessibility Insights">
+</figure>
+
+## Don't convey information with color alone
+
+There are roughly 320 million people worldwide with color vision deficiency.
+About 1 in 12 men and 1 in 200 women have some form of "color blindness";
+that means about 1/20th, or 5%, of your users will not experience your site the way you intended.
+When we rely on color to convey information, we push that number to unacceptable levels.
+
+{% Aside %}
+Note: The term "color blindness" is often used to describe a visual condition where a person has trouble distinguishing colors,
+but in fact very few people are truly color blind.
+Most people with color deficiencies can see some or most colors,
+but have difficulty differentiating between certain colors such as reds and greens (most common),
+browns and oranges, and blues and purples.
+{% endAside %}
+
+For example, in an input form,
+a telephone number might be underlined in red to show that it is invalid.
+But to a color deficient or screen reader user, that information is not conveyed well, if at all.
+Thus, you should always try to provide multiple avenues for the user to access critical information.
+
+<figure class="w-figure">
+  <img src="./input-form1.png" alt="A image of an input form with an incorrect phone number highlighted only with a red color.">
+</figure>
+
+The [WebAIM checklist states in section 1.4.1](https://webaim.org/standards/wcag/checklist#sc1.4.1) that
+"color should not be used as the sole method of conveying content or distinguishing visual elements."
+It also notes that "color alone should not be used to distinguish links from surrounding text"
+unless they meet certain contrast requirements.
+Instead, the checklist recommends adding an additional indicator such as an underscore
+(using the CSS text-decoration property) to indicate when the link is active.
```

Comment:
```suggestion
(using the CSS `text-decoration` property) to indicate when the link is active.
```
In general whenever you see a reference to a CSS property or JavaScript variable please format as inline code